### PR TITLE
Feat: Make relevant strings non-translatable

### LINKED ISSUE #944: Feature Request: Add translatable=false flag to make translations easier to do
**Summary:**

There are some strings which shouldn't be translated.

**Steps to reproduce:**

app>src>main>res>values>strings.xml

**Expected behavior:**

Add translatable="false" flag to non translatable strings.

**Observed behavior:**

There are some strings which shouldn't be translated, but these don't have the flag.

### PR DESCRIPTION
**Pull request by m-sameer**
Fixes #944

The build has been fixed in #959 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -504,7 +504,7 @@
 
     <string name="pref_base_url_title">Update Endpoint</string>
     <string name="pref_base_url_desc">Click here to change your Endpoint Configurations</string>
-    <string name="pref_configuration">pref_configuration</string>
+    <string name="pref_configuration" translatable="false">pref_configuration</string>
```
Comment (miPlodder) [inline app/src/main/res/values/strings.xml:507]: Remove these lines from other translations files.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -355,8 +355,8 @@
     <string name="select_region_qr">Select region with QR code</string>
     <!--Customer care details-->
     <string name="need_help">Contact Us</string>
-    <string name="help_line_number">8000000000</string>
-    <string name="contact_email">support@mifos.org</string>
+    <string name="help_line_number" translatable="false">8000000000</string>
+    <string name="contact_email" translatable="false">support@mifos.org</string>
```
Comment (miPlodder) [inline app/src/main/res/values/strings.xml:359]: Remove these lines from other translations files.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -355,8 +355,8 @@
     <string name="select_region_qr">Select region with QR code</string>
     <!--Customer care details-->
     <string name="need_help">Contact Us</string>
-    <string name="help_line_number">8000000000</string>
-    <string name="contact_email">support@mifos.org</string>
+    <string name="help_line_number" translatable="false">8000000000</string>
```
Comment (miPlodder) [inline app/src/main/res/values/strings.xml:358]: Remove these lines from other translations files.

**Review by miPlodder — CHANGES_REQUESTED** [review]

**Comment by therajanmaurya** [comment]
@m-sameer Please make suggested changes.

**Comment by prashantkh19** [comment]
#1179 fixes this. @therajanmaurya close this.

**Pull request closed without merging**